### ISSUE TITLE
Code action widget accessibility strangeness

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#162126]
![Recording 2022-09-28 at 09 13 49](https://user-images.githubusercontent.com/900690/192712837-0eee8b18-fe59-4cde-adf1-ed7b092598ad.gif)

I find it a bit strange that hovering over entries in the list puts the blue focus color and then pressing enter seems to take the element under the mouse. I think that is not very accessible or expected, I would have rather expected the blue element to stay stable and only change with keyboard up/down.

//cc @isidorn @daviddossett

**Comment by isidorn** [comment]
Agree. Would be great if the navigation + hover pattern align with existing UI we have.
